arm-advisor/2016-07-12-preview/swagger/advisor.json

```diff
@@ -0,0 +1,641 @@
+				"operationId": "Suppressions_Delete",
+				"parameters": [{
+					"name": "resourceUri",
+					"in": "path",
+					"description": "The fully qualified arm identifier of the resource for which the recommendation applies.",
+					"required": true,
+					"type": "string"
+				},
+				{
+					"name": "recommendationId",
+					"in": "path",
+					"description": "The recommendation Id",
+					"required": true,
+					"type": "string"
+				},
+				{
+					"name": "name",
+					"in": "path",
+					"description": "The suppression name.",
+					"required": true,
+					"type": "string"
+				},
+				{
+					"$ref": "#/parameters/apiVersionParameter"
+				}],
+				"responses": {
+					"204": {
+						"description": "The recommendation has been activated."
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"DeleteSuppression": {
+						"$ref": "../examples/DeleteSuppression.json"
+					}
+				}
+			}
+		},
+		"/{resourceUri}/providers/Microsoft.Advisor/recommendations/{recommendationId}": {
+			"get": {
+				"tags": ["GetRecommendations"],
+				"description": "Obtains details of a cached recommendation.",
+				"operationId": "Recommendations_Get",
+				"parameters": [{
+					"name": "resourceUri",
+					"in": "path",
+					"description": "The fully qualified arm identifier of the resource for which the recommendation applies.",
+					"required": true,
+					"type": "string"
+				},
+				{
+					"name": "recommendationId",
+					"in": "path",
+					"description": "The recommendation Id.",
+					"required": true,
+					"type": "string"
+				},
+				{
+					"$ref": "#/parameters/apiVersionParameter"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResourceRecommendationBase"
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"GetRecommendationDetail": {
+						"$ref": "../examples/GetRecommendationDetail.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/suppressions": {
+			"get": {
+				"tags": ["Suppressions"],
+				"summary": "",
+				"description": "Retrieves the list of snoozed/dismissed suppressions for a subscription. The snooze/dismiss attribute of a recommendation is referred to as a suppression.",
+				"operationId": "Suppressions_List",
+				"parameters": [{
+					"$ref": "#/parameters/subscriptionIdParameter"
+				},
+				{
+					"$ref": "#/parameters/apiVersionParameter"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "array",
+							"items": {
+								"$ref": "#/definitions/Suppression"
+							}
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"ListSuppressions": {
+						"$ref": "../examples/ListSuppressions.json"
+					}
+				}
+			}
+		}
+	},
+	"definitions": {
+		"CollectionResponse[ResourceRecommendationBase]": {
```

Comment:
I believe you are generating this from swashbuckle. Swagger file should not have `[`, please configure it correctly.